clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -888,7 +886,7 @@ public void enable() {
 
         public void disable() {
             synchronized (AbstractCoordinator.this) {
-                log.trace("Disabling heartbeat thread for group {}", groupId);
+                log.debug("Disabling heartbeat thread");
```

Comment:
Also worth mentioning this in the PR description with regards to changes in log level.